tasks.py

```diff
@@ -70,8 +70,8 @@ def _validate_python_version(line):
 
 @task
 def install_minimum(c):
-    with open('setup.py', 'r') as setup_py:
-        lines = setup_py.read().splitlines()
+    with open('pyproject.toml', 'r') as pyproject:
```

Comment:
Does this code work? For getting the minimum dependencies from `pyproject.toml` you can do something like the following:
```python
import tomli

def parse_pyproject_toml(path):
    requirements = []
    with open(paths, "rb") as f:
        toml_dict = tomli.load(f)

    requirements = toml_dict["project"]["dependencies"]
    optional_reqs = toml_dict["project"]["optional-dependencies"]["all"]
    return requirements, optional_reqs

def clean_list_length_one(item):
    if isinstance(item, list) and len(item) == 1 and " " in item[0]:
        item = item[0].split(" ")
    if item == [""]:
        return None
    return item

requirements = parse_pyproject_toml('pyproject.toml')
```

Reply:
Yes it works. I tried to make the minimum changes here. In commit [5919e05](https://github.com/sdv-dev/SDGym/pull/269/commits/5919e05887c2179fee67aa2e8b5b7d67636675ad) I was doing something similar as your proposal, but this add a dependency (`toml` in my case), so let me know if we can add a dependency.

Reply:
Yep, my vote would that it is okay to add this dependency (it's only generating minimum dependencies).

Reply:
Alright, done in [3ef342a](https://github.com/sdv-dev/SDGym/pull/269/commits/3ef342a60d307271038e824a25ea77a75876712f)